index.html

```diff
@@ -402,8 +421,8 @@ <h2 class="contact-me" id="contact">Contact me</h2>
           that you need built or a project that needs coding. I'd love to help
           with it
         </p>
```

Comment:
- [ ] Great job so far, But I would like to Highlight that here you need to align the form in the center and you should decrease the width for each input and align them all in the center so you can use flex for that it would be amazing 💪🏻💪🏻so please Fix that and center the form as well to match the Figma design. 💪🏻💪🏻

- [ ] please Adjust the background to match the design as much as possible 💥💥
- your design 
![image](https://user-images.githubusercontent.com/58553711/177785464-a4795038-1f57-45de-8c29-cccfa652d91c.png)

- Figma design 
![image](https://user-images.githubusercontent.com/58553711/177785563-bf42c96a-24e0-4db7-86b6-dacac5d886c1.png)